app/lib/annotation_submission_helper.js

```diff
@@ -259,20 +262,27 @@ function verifyCommentFields(annotation, locusMap) {
 function createGeneTermRecords(annotation, locusMap, transaction) {
 	let subAnnotation = {
 		method_id: annotation.data.method.id,
-		keyword_id: annotation.data.keyword.id
+		keyword_id: annotation.data.keyword.id,
 	};
 
-	// 'evidence' is an optional field
-	if (annotation.data.evidence) {
-		subAnnotation.evidence_id = locusMap[annotation.data.evidence].locus.get('locus_id');
-
-		// GeneSymbols are optional
-		if (locusMap[annotation.data.evidence].symbol) {
-			subAnnotation.evidence_symbol_id = locusMap[annotation.data.evidence].symbol.get('id');
-		}
-	}
-
-	return GeneTermAnnotation.forge(subAnnotation).save(null, {transacting: transaction});
+	return GeneTermAnnotation.forge(subAnnotation).save(null, {transacting: transaction})
+        .then( (geneTermAnnotation) => {
+            //TODO: check if not locus and act accordingly
+            // add evidence_with to db if exists
+            let evidenceWithPromises = [];
+             
+            for(const subject of annotation.data.evidenceWith) {
+                evidenceWithPromises.push(EvidenceWith.forge({
+                    subject_id: locusMap[subject].locus.get('locus_id'),
+                    gene_term_id: geneTermAnnotation.id,
+                    type: 'temp' //TODO: actually implement type
```

Comment:
This needs to be Locus or rather look into how Bookshelf/ other code creates morph relations.